pkg/gadgettracermanager/api/gadgettracermanager.proto

```diff
@@ -83,6 +83,9 @@ message ContainerDefinition {
   string cgroup_v1 = 9;
   string cgroup_v2 = 10;
   repeated string mount_sources = 11;
+
+  // Data used to change network namespaces
+  uint32 container_pid = 12;
```

Comment:
If we want to remove the `container_` prefix (change `container_id` to `id`), we can do that because the only user of the grpc interface is Inspektor Gadget and it's only local/internal communication, so there is no compatibility concern.

Reply:
You're right, `container_` prefix is not implicit here. I removed it from both members.

Reply:
Mmmm I was removing also `container_` prefix from `ContainerDefinition.container_name` but it is usually compared against `ContainerSelector` where it is also named `container_name`. In addition, I noticed there is also `ContainerFilter.container_name`. 

At this point, I think that it is better to change all of them to just `name` or none. What do you think?

Reply:
I think we can change all of them.